main.go

```diff
@@ -85,7 +85,8 @@ func main() {
 	}
 
 	ctrlOptions := ctrl.Options{
-		Scheme: scheme,
+		Scheme:             scheme,
+		MetricsBindAddress: "0", // disable metrics
```

Comment:
cant u pick other port on line 68 ?

Reply:
Please add the comment to the proper line, I have remove the argument all together so no need to do anything about it.